### PR TITLE
Adding docs/RN for all versions

### PR DESCRIPTION
**Pull request by jtravee**

### CODE REVIEW

File: src/pages/versions.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  <tr>
+    <td><b>v2.5.16</b></td>
+    <td><a href="https://rancher.com/docs/rancher/v2.5/en/">Documentation</a></td>
+    <td><a href="https://github.com/rancher/rancher/releases/tag/v2.5.16">Release Notes</a></td>
+  </tr>
+</table>
+
+<br/>
+
+### Past versions (no longer maintained)
```
Comment (btat) [inline src/pages/versions.md:29]:
```suggestion
### Past versions
```

File: src/pages/versions.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
+---
+title: Rancher Documentation Versions
+---
+
+### Current versions (stable)
```
Comment (btat) [inline src/pages/versions.md:5]:
```suggestion
### Current versions
```

File: src/pages/versions.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    <td><a href="https://github.com/rancher/rancher/releases/tag/v2.6.8">Release Notes</a></td>
+  </tr>
+</table>
+
+Below are the documentation and release notes for the currently released version of Rancher 2.5.x:
+
+<table>
+  <tr>
+    <td><b>v2.5.16</b></td>
+    <td><a href="https://rancher.com/docs/rancher/v2.5/en/">Documentation</a></td>
```
Comment (btat) [inline src/pages/versions.md:22]:
```suggestion
    <td><a href="https://URL_PLACEHOLDER/v2.5/rancher-manager">Documentation</a></td>
```

File: src/pages/versions.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+<br/>
+
+### Legacy versions (EOL)
+
+Below is the documentation for legacy versions of Rancher 2.0 - 2.4.x:
+
+<table>
+  <tr>
+    <td><b>v2.0 - v2.4</b></td>
+    <td><a href="https://rancher.com/docs/rancher/v2.0-v2.4/en/">Documentation</a></td>
```
Comment (btat) [inline src/pages/versions.md:172]:
```suggestion
    <td><a href="https://URL_PLACEHOLDER/v2.5/rancher-manager">Documentation</a></td>
```

File: src/pages/versions.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+---
+
+### Current versions (stable)
+
+Below are the documentation and release notes for the currently released version of Rancher 2.6.x:
+
+<table>
+  <tr>
+    <td><b>v2.6.8</b></td>
+    <td><a href="https://rancher.com/docs/rancher/v2.6/en/">Documentation</a></td>
```
Comment (btat) [inline src/pages/versions.md:12]:
```suggestion
    <td><a href="https://URL_PLACEHOLDER/">Documentation</a></td>
```

**Review by btat — CHANGES_REQUESTED** [review]
The stable/unmaintained labels should be removed because of how our versions are structured. E.g. For 2.6.x the docs are either all maintained or all unmaintained, but it wouldn't be the case we maintain 2.6.8, but not earlier because our setup doesn't differentiate.

I've added examples of showing the how the known portions of the URLs will be formatted. Those will need to be updated for each entry.

Another column we could link to is the Support Matrix. Thoughts on adding that?

File: src/pages/versions.md (hunk 1 above)
Comment (jtravee) [reply src/pages/versions.md]: Updated.

File: src/pages/versions.md (hunk 2 above)
Comment (jtravee) [reply src/pages/versions.md]: Updated.

File: src/pages/versions.md (hunk 3 above)
Comment (jtravee) [reply src/pages/versions.md]: Updated.

File: src/pages/versions.md (hunk 5 above)
Comment (jtravee) [reply src/pages/versions.md]: Updated.

**Comment by jtravee** [comment]
> The stable/unmaintained labels should be removed because of how our versions are structured. E.g. For 2.6.x, the docs are either all maintained or all unmaintained, but it wouldn't be the case we maintain 2.6.8, but not earlier because our setup doesn't differentiate. So it doesn't make sense to use the stable/maintained labels.
> 
> I've added examples of showing the how the known portions of the URLs will be formatted. Those will need to be updated for each entry.
> 
> Another column we could link to is the Support Matrix. Thoughts on adding that?

Great idea on the support matrix; I'll add those.

**Review by btat — CHANGES_REQUESTED** [review]
Now that we have a confirmed URL, could you update the placeholders.

**Review by btat — CHANGES_REQUESTED** [review]
The wrong URL was added. Should be `http://docs.ranchermanager.rancher.io/`.

Also, looks like parts of the URL got left off for the older versions.
E.g. /v2.5/ and /v2.0-v2.4/

**Review by btat — APPROVED** [review]

**Pull request merged**